apps/frontend/core-dapp/src/features/portfolio/PositionsAndHistoryNext.tsx

```diff
@@ -0,0 +1,100 @@
+import { t } from '@lingui/macro'
+import { Box, centered, media, spacingIncrement } from 'prepo-ui'
+import { useState } from 'react'
+import styled, { useTheme } from 'styled-components'
+import FilterButton from '../../components/Filter/FilterButton'
+import Tabs from '../../components/Tabs'
+import History from '../history/History'
+import Positions from '../position/PositionsNext'
+
+const StyledTabs = styled(Tabs)`
+  &&&& {
+    .ant-select-selection-item {
+      padding-right: 20px;
+    }
+
+    .ant-select-selection-item,
+    .ant-tabs-tab-btn {
+      font-size: ${({ theme }): string => theme.fontSize.lg};
+    }
+
+    .ant-select-arrow {
+      left: 100%;
+      padding: 0 ${spacingIncrement(6)};
+      transform: translate(-100%, -50%);
+    }
+
+    .ant-tabs-nav {
+      flex-direction: column;
+      margin-bottom: 0;
+      ${media.desktop`
+        flex-direction: row;
+      `}
+    }
+
+    .ant-tabs-extra-content {
+      ${media.desktop`
+        width: auto;
+        padding: 0 ${spacingIncrement(30)};
+      `}
+      align-items: flex-end;
+      display: flex;
+      justify-content: flex-end;
+      padding: ${spacingIncrement(20)} ${spacingIncrement(37)};
+      width: 100%;
+    }
+
+    .ant-tabs-tab {
+      display: flex;
+      flex: 1;
+      padding: ${spacingIncrement(8)} 0;
+      width: 100%;
+    }
+
+    .ant-tabs-nav-list {
+      display: grid;
+      gap: ${spacingIncrement(30)};
+      grid-template-columns: auto auto;
+      padding: 0 ${spacingIncrement(24)};
+    }
+
+    .ant-tabs-nav-list,
+    .ant-tabs-tab,
+    .ant-tabs-tab .ant-tabs-tab-active {
+      ${centered}
+    }
+
+    .ant-tabs-ink-bar {
+      border-radius: 1px 1px 0 0;
+      height: ${spacingIncrement(4)};
+    }
```

Comment:
Offset 4px from tab item

```suggestion
    .ant-tabs-ink-bar {
      border-radius: 1px 1px 0 0;
      height: ${spacingIncrement(4)};
      bottom: ${spacingIncrement(-4)};
    }
```

Reply:
I'm not following why do we need this. I made the changes on 409cd2b0425ec87622d58809b51c2b3eba4d8dd9 and it looks fine to me:

![Screenshot 2023-02-07 at 09 04 17](https://user-images.githubusercontent.com/11708035/217186969-00276d64-6ff1-4725-b3d7-af87231fb318.png)

Btw, off topic, but I subscribe to the opinion that [shadows should be dark on dark mode](https://twitter.com/danqing_liu/status/1579536100980592641), what do you think?